test/components/alpha/auto_complete/item_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "test_helper"
+class PrimerAutoCompleteItemTest < Minitest::Test
+  include Primer::ComponentTestHelpers
```

Comment:
I'm surprised the linter isn't catching this. There should be a newline after this `include`.